apps/content/migrations/0028_alter_reciter_date_of_birth_and_more.py

```
@@ -0,0 +1,33 @@
+# Generated by Django 5.2.11 on 2026-02-24 18:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("content", "0027_reciter_reciter_identifier"),
+    ]
```

Ruff (0.15.2)

[warning] Mutable default value for class attribute

(RUF012)